pxtlib/tutorialValidator.ts

```diff
@@ -5,6 +5,7 @@ namespace pxt.tutorial {
         RuleTurnOn: boolean;
         RuleStatus: boolean;
         RuleMessage: string;
+        isStrict: boolean;
```

Comment:
for later; it might be worth considering leaving some of these as optional (e.g. `isStrict?: boolean`); on line 64 down below for example, RuleStatus, RuleMessage, and isStrict don't appear to necessarily have full values set yet / it looks like they're only being filled in to satisfy the compiler, whereas leaving them undefined might make it easier to identify in the future (having false or "" explicitly set can sometimes feel like it's intentional, where undefined is pretty clear it's missing)